content/en/docs/concepts/configuration/secret.md

```diff
@@ -511,7 +571,7 @@ The name of a Secret object must be a valid
 [DNS subdomain name](/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
 
 You can specify the `data` and/or the `stringData` field when creating a
-configuration file for a Secret. The `data` and the `stringData` fields are optional.
+configuration file for a Secret.  The `data` and the `stringData` fields are optional.
```

Comment:
Even better:
```suggestion
manifest for a Secret.  The `data` and the `stringData` fields are optional.
```

Reply:
Yes but out of scope I think for this PR, although it probably looked like it was touched because of my horrible rebase earlier. This page says "configuration file" instead of "manifest" often :(